pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -72,22 +94,54 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Replica Quiescence" sources={storeSources}>
+    <LineGraph
+      title="Replica Quiescence"
+      sources={storeSources}
+      tooltip={(
+        <div>
+        Number of replicas and quiesced replicas {tooltipSelection}.
+        <br/>
+        Quiesced replicas have no pending reads or writes and have temporarily disabled consensus protocols.
+        </div>
+      )}
+    >
       <Axis label="replicas">
         <Metric name="cr.store.replicas" title="Replicas" />
         <Metric name="cr.store.replicas.quiescent" title="Quiescent" />
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Range Operations" sources={storeSources}>
+    <LineGraph title="Range Operations" sources={storeSources}
+      tooltip = {`Ranges with split, add, or remove operations ${tooltipSelection}.`}
```

Comment:
I think this is counting the number of operations, not the number of ranges with those operations.  For the most part it would be the same, but not always.  For instance, if a range is split twice, we expect it to be counted twice here.

also please no spaces around the `=`.  surprised that our linter didn't complain about this, but i think the jsx checking is kind of lax.